core/admin/Dockerfile

```diff
@@ -26,7 +26,7 @@ WORKDIR /app
 COPY requirements-prod.txt requirements.txt
 RUN apk add --no-cache libressl curl postgresql-libs mariadb-connector-c \
   && apk add --no-cache --virtual build-dep \
-  libressl-dev libffi-dev python3-dev build-base postgresql-dev mariadb-connector-c-dev \
+  libressl-dev libffi-dev python3-dev build-base postgresql-dev mariadb-connector-c-dev cargo \
```

Comment:
Is there a specific reason why are you adding the cargo module?
Is this a new dependency for the updated modules in requirements-prod.txt?

Reply:
Not my code but it's to ensure that the rust wheels get built properly; without there was a warning (and either the wheels wouldn't be built or it would be downloaded at build-time)

Reply:
Yes. It seems to be possible to disable the rust part via an environment variable, but only for cryptography 3.4:
"You can disable Rust integration in 3.4.x (see FAQ). Starting with 3.5 cryptography will have mandatory Rust code."
Enabling it now seemed to be more future proof.

Reply:
It's a bad idea to not have the wheels; an attacker will have an optimized runtime, so should we :)

This is useful for (fast) iterated password hashing